src/app/AttributePersistenceProvider.h

```diff
@@ -15,8 +15,13 @@
  */
 #pragma once
 
+#include <app-common/zap-generated/attribute-type.h>
```

Comment:
Does the top-level EmberAfAttributeType live anywhere not code-generated?

Reply:
Not that I can see. I am only importing this because of `ZCL_INT8U_ATTRIBUTE_TYPE`. I can hard-code this value and not import this file and put a note next to it.

Reply:
OK. Do what you think is best.

Reply:
I removed it.